controllers/feeds/getFeedsV2.js

```diff
@@ -117,14 +113,17 @@ module.exports = async (req, res) => {
       listBlockUser,
       listBlockDomain,
       listPostAnonymousAuthor,
-      isBlurredPost
+      isBlurredPost,
+      allFollowingUser
     ] = await Promise.all([
       UsersFunction.findAnonymousUserId(User, req.userId, {raw: true}),
       getListBlockUser(req.userId),
       getBlockDomain(req.userId),
       getListBlockPostAnonymousAuthor(req.userId),
-      UserFollowUserFunction.checkIsBlurredPost(UserFollowUser, req.userId)
+      UserFollowUserFunction.checkIsBlurredPost(UserFollowUser, req.userId),
+      UserFollowUserFunction.getAllFollowingUser(UserFollowUser, req.userId)
     ]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `allFollowingUser` to the Promise.all array is a good way to fetch all necessary data concurrently. However, ensure that `UserFollowUserFunction.getAllFollowingUser` function handles errors properly and doesn't cause the entire Promise.all to reject if it fails.

```diff
      listBlockDomain,
      listPostAnonymousAuthor,
-     isBlurredPost
+     isBlurredPost,
+     allFollowingUser
     ] = await Promise.all([
       UsersFunction.findAnonymousUserId(User, req.userId, {raw: true}),
       getListBlockUser(req.userId),
       getBlockDomain(req.userId),
       getListBlockPostAnonymousAuthor(req.userId),
      UserFollowUserFunction.checkIsBlurredPost(UserFollowUser, req.userId),
+     UserFollowUserFunction.getAllFollowingUser(UserFollowUser, req.userId)
     ]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->